src/js/bootstrap-slider.js

```diff
@@ -185,11 +185,10 @@ const windowIsDefined = (typeof window === "object");
 			        return $(this);
 			      });
 
-			      if(!objects || objects.length > 1) {
-			      	return objects;
-			      } else {
+			      if(objects.length === 1) {
```

Comment:
Great work and the JSFiddle is always helpful.

Just a couple of things.

1. We only need changes to the source `bootstrap-slider.js` file in `src/js`.
1. You can discard the changes to the distribution files (everything under `dist/` folder) because those files are generated when a release is ready to be published to npm
1. Lastly, can you add a test in `test/specs/PublicMethodsSpec.js` file for this specific issue to prevent future releases/patches from reintroducing this bug?

**Note:** The check `!objects` is not necessary because `.map(callback)` will always return a jQuery object.

Reply:
Okay
- I've reverted the changes to the `dist` folder back to the `v10.6.1` tag
- I've added tests in  `test/specs/PublicMethodsSpec.js` to make sure that when called on a jQuery object with zero or more matches it will return a jQuery object.